torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1470,6 +1499,188 @@ def _make_tensor_helper(shape, low=None, high=None):
 
     return samples
 
+foreach_unary_op_db: List[OpInfo] = [
+    ForeachUnaryFuncInfo('_foreach_neg',
+                         method=torch._foreach_neg,
+                         inplace=torch._foreach_neg_,
+                         ref=torch.neg,
+                         dtypes=all_types_and_complex(),
+                         dtypesIfCPU=all_types_and_complex(),
+                         dtypesIfCUDA=all_types_and_complex(),
+                         sample_inputs_func=sample_inputs_foreach,
+                         safe_casts_outputs=False),
+
+    ForeachUnaryFuncInfo('_foreach_sqrt',
+                         method=torch._foreach_sqrt,
```

Comment:
Definitely not a blocker for this PR, but one thing we do with the base OpInfos is attempt to programmatically acquire the method and inplace variants of an op, which this could do, too, if they weren't specified.

Reply:
done